tests_e2e/pipeline/scripts/execute_tests.sh

```diff
@@ -54,10 +55,6 @@ if [[ $VM_SIZE == "-" ]]; then
     VM_SIZE=""
 fi
 
-# A test failure will cause automation to exit with an error code and we don't want this script to stop so we force the command
-# to succeed and capture the exit code to return it at the end of the script.
-echo "exit 0" > /tmp/exit.sh
```

Comment:
So, the time outs not captured here in exit?

Reply:
azure pipelines kills the process after 60 min so the code to collect logs is not executed. the PR moves that code to a different step, which will be executed after this one is killed